docs/tutorials/hello_world/README.md

```diff
@@ -225,6 +235,175 @@ query {
 }
 ```
 
+We can use graphql to search for events by name and version.
+
+Create another event for the same event receiver
+
+```graphql
+mutation {
+  create_event(
+    event: {
+      name: "foo"
+      version: "1.0.0"
+      release: "20240217"
+      platform_id: "x86-64-gnu-linux-7"
+      package: "rpm"
+      description: "The RPM Foo of Brixton"
+      payload: "{\"name\": \"foo\"}"
+      event_receiver_id: "01HPVZY1V8SVXGQY03ZG90CA3S"
+      success: true
+    }
+  )
+}
+```
+
+In the graphql window create a query with the following:
+
+```graphql
+query ($e: FindEventInput!) {
+  events(event: $e) {
+    id
+    name
+    version
+    release
+    platform_id
+    package
+    description
+    payload
+    success
+    event_receiver_id
+    created_at
+  }
+}
+```
+
+In the variables section, we need to pass in the name and version.
+
+```grapql
+{
+  "e": {
+    "name":  "foo",
+    "version": "1.0.0"
+  }
+}
+```
```

Comment:
Can be simplified to:

```suggestion
```graphql
query {
  events(event: {
    "name":  "foo",
    "version": "1.0.0"
  }) {
    id
    name
    version
    release
    platform_id
    package
    description
    payload
    success
    event_receiver_id
    created_at
  }
```